src/StaticFAQ/index.js

```diff
@@ -45,22 +47,36 @@ type Props = {|
 type State = {|
   searchText: string,
 |};
-
+const logSearchQuery = debounce(q =>
+  simpleTracker('smartFAQBookingOverview', {
+    action: 'search',
+    searchedText: q,
+  }),
+);
 class StaticFAQ extends React.Component<Props, State> {
   changeSearchText = (text: string) => {}; // eslint-disable-line no-unused-vars
+  resetQueriesCount = () => {}; // eslint-disable-line no-unused-vars
+  incrementQueriesCount = () => {}; // eslint-disable-line no-unused-vars
+
   handleSearchChange = (e: SyntheticInputEvent<HTMLInputElement>) => {
     this.changeSearchText(e.target.value);
+    logSearchQuery(e.target.value);
   };
   handleCancelSearch = () => {
     this.changeSearchText('');
+    this.resetQueriesCount();
   };
 
   renderInput = (searchText: string) => {
     const isSearching = searchText.length > 0;
+    const inputChange = e => {
```

Comment:
doing it in `handleSearchChange` should be fine enough, I guess there is no reason to define new function that wraps the previous one?